src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('.field');
+const rows = table.rows;
+
+const addRowButton = document.querySelector('.append-row');
+const addColumnButton = document.querySelector('.append-column');
+const removeRowButton = document.querySelector('.remove-row');
+const removeColumnButton = document.querySelector('.remove-column');
+
+addRowButton.addEventListener('click', (e) => {
+  removeRowButton.disabled = false;
+  table.append(rows[0].cloneNode(true));
+
+  if (rows.length > 9) {
```

Comment:
it is better not use "magic" numbers. 
Create a constant for this 
Example: `const MAX_LENGTH = 9`

Reply:
Thanks, it makes code much more readable, indeed.